src/Features/Core/Portable/CodeFixesAndRefactorings/CodeActionRequestPriorityProvider.cs

```diff
@@ -15,7 +15,10 @@ namespace Microsoft.CodeAnalysis.CodeActions
 {
     internal interface ICodeActionRequestPriorityProvider
     {
-        CodeActionRequestPriority Priority { get; }
+        /// <summary>
+        /// <see langword="null"/> represents no specified priority.  i.e. any priority should match this.
+        /// </summary>
+        CodeActionRequestPriority? Priority { get; }
```

Comment:
we used to have CodeActionRequestPriority.None.  But that was a super wonky thing to put in the interface.  The new way, which makes things much cleaner is to have these internal APIs just use 'null' to indicate they match all priorities.